indico/MaKaC/badgeDesignConf.py

```diff
@@ -132,34 +75,44 @@ class BadgeDesignConfiguration:
-                               "City":                    (_("City"), Registrant.getCity),
-                               "Address":                 (_("Address"), Registrant.getAddress),
-                               "Phone":                   (_("Phone"), Registrant.getPhone),
-                               "Fax":                     (_("Fax"), Registrant.getFax),
-                               "Email":                   (_("Email"), Registrant.getEmail),
-                               "Personal homepage":       (_("Personal homepage"), Registrant.getPersonalHomepage),
-                               "Amount":                  (_("Amount"), Registrant.getTotal),
-                               "Conference Name":         (_("Conference Name"), Conference.getTitle),
-                               "Conference Dates":        (_("Conference Dates"), ConferenceDates),
-                               "Fixed Text":              (_("Fixed Text"), BadgeTemplateItem.getFixedText)
-                            }
-
-            """ Dictionary that maps group names to the item names that fall into that group.
-            The groups are only used for the <select> box in the WConfModifBadgeDesign.tpl file.
-            """
-            self.groups = [( _("Registrant Data"), [ "Title", "Full Name", "Full Name (w/o title)", "Full Name B", "Full Name B (w/o title)", "Full Name C", "Full Name C (w/o title)", "First Name",  "Surname",  "Position",
-                                                     "Institution",  "Country",  "City", "Address", "Phone", "Fax", "Email", "Personal homepage", "Amount"]),
-                      ( _("Conference Data"), [ "Conference Name",  "Conference Dates"]),
-                      ( _("Fixed Elements"), [ "Fixed Text"])]
+        title_lambda = lambda x: x.get_personal_data()['title']
+        full_name_lambda = lambda x: "{} {} {}".format(x.get_personal_data()['title'], x.last_name, x.first_name)
+        full_name_no_title_lambda = lambda x: "{} {}".format(x.last_name, x.first_name)
+        full_name_name_first_lambda = lambda x: "{} {} {}".format(x.get_personal_data()['title'], x.first_name,
+                                                                  x.last_name)
+        full_name_name_first_no_title = lambda x: "{} {}".format(x.first_name, x.last_name)
+        first_name_lambda = lambda x: x.first_name
+        last_name_lambda = lambda x: x.last_name
+        affiliation_lambda = (lambda x: x.get_personal_data()['affiliation']
+                              if 'affiliation' in x.get_personal_data() else '')
+        address_lambda = lambda x: x.get_personal_data()['address'] if 'address' in x.get_personal_data() else ''
+        phone_lambda = lambda x: x.get_personal_data()['phone'] if 'phone' in x.get_personal_data() else ''
+        email_lambda = lambda x: x.email
+        price_lambda = lambda x: x.price
+
+        self.items_actions = { "Title": (_("Title"), title_lambda),
```

Comment:
a wild space appeared (or stayed). I'd wrap after `{` to have more space for the lambdas